src/components/blocks/software/Pre.tsx

```diff
@@ -38,6 +38,15 @@ const Pre = ({
 }: PreProps): ReactElement => {
   const pageLanguage = useContext(PageLanguageContext);
 
+  /*  selectedInterfaceTab useState  */
+  const [selectedSDKInterfaceTab, setSelectedSDKInterfaceTab] = useState(DEFAULT_PREFERRED_INTERFACE);
+  const [previousSDKInterfaceTab, setPreviousSDKInterfaceTab] = useState('');
+
+  /* only pass the languages that are SDK interface active */
```

Comment:
what does this mean?

Reply:
this is for the changing of Rest and Realtime tab state
also need save the value of  previous tab in state